CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
 
-## 0.55.1
+## 0.56.0
```

Comment:
Not sure what `semver` specifies in this case. We are in 0.x and we can do whatever but at the end we are fixing something not adding nothing more. As long as semver allows breaking changes in 0.x does not mean that a fix ends up in a feature, or am I missing something?